charts/descheduler/templates/servicemonitor.yaml

```
@@ -7,6 +7,9 @@ metadata:
   namespace: {{ .Values.serviceMonitor.namespace | default .Release.Namespace }}
   labels:
     {{- include "descheduler.labels" . | nindent 4 }}
+    {{- if .Values.serviceMonitor.additionalLabels }}
+    {{ toYaml .Values.serviceMonitor.additionalLabels }}
```

shouldn't this be indented?
```suggestion
    {{ toYaml .Values.serviceMonitor.additionalLabels | nindent 4 }}
```